packages/m2m-gateway/test/api/tenants/getTenants.test.ts

```diff
@@ -0,0 +1,105 @@
+import { describe, it, expect, vi } from "vitest";
+import { generateToken } from "pagopa-interop-commons-test";
+import { AuthRole, authRole } from "pagopa-interop-commons";
+import request from "supertest";
+import { m2mGatewayApi } from "pagopa-interop-api-clients";
+import { api, mockTenantService } from "../../vitest.api.setup.js";
+import { appBasePath } from "../../../src/config/appBasePath.js";
+import { toM2MGatewayApiTenant } from "../../../src/api/tenantApiConverter.js";
+import { getMockedApiTenant } from "../../mockUtils.js";
+
+describe("GET /tenants route test", () => {
+  const mockApiTenant1 = getMockedApiTenant();
+  const mockApiTenant2 = getMockedApiTenant();
+
+  const mockResponse: m2mGatewayApi.Tenants = {
+    results: [
+      toM2MGatewayApiTenant(mockApiTenant1.data),
+      toM2MGatewayApiTenant(mockApiTenant2.data),
+    ],
+    pagination: {
+      limit: 10,
+      offset: 0,
+      totalCount: 2,
+    },
+  };
+
+  const mockQueryParams: m2mGatewayApi.GetTenantsQueryParams = {
+    externalIdOrigin: undefined,
+    externalIdValue: undefined,
+    offset: 0,
+    limit: 10,
+  };
+
+  const makeRequest = async (token: string, query: Record<string, unknown>) =>
+    request(api)
+      .get(`${appBasePath}/tenants`)
+      .query(query)
+      .set("Authorization", `Bearer ${token}`);
+
+  const authorizedRoles: AuthRole[] = [
+    authRole.M2M_ROLE,
+    authRole.M2M_ADMIN_ROLE,
+  ];
+
+  it.each(authorizedRoles)(
+    "Should return 200 and perform API clients calls for user with role %s",
```

Comment:
Could you take advantage of this PR to update this also in all other **API** tests merged in #1848 and #1832 

```suggestion
    "Should return 200 and perform service calls for user with role %s",
```